client/src/DeleteButton.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import axios from 'axios'
+import { Button, Modal } from 'react-bootstrap'
+
+
+class DeleteButton extends React.Component {
+	constructor (props) {
+		super(props)
+		this.state = {
+			showDelete: false
+		}
+		this.handleDelete = this.handleDelete.bind(this)
+	}
+
+	// Handlers for delete button
+	handleClose = () => {this.setState({showDelete: false})};
+	handleShow = () => {this.setState({showDelete: true})};
+	handleDelete (e) {
+		axios.delete(`/api${this.props.location.pathname}`)
+		.then(res => {
+			console.log('got to delete.then')
+			this.props.history.push({ pathname: '/'})
+		})
+	}
+
+
+	render() {
+		return (
+			<div>
+          <Button variant="danger" onClick={this.handleShow}>
```

Comment:
[nit] This indentation still hella wack lmao

Reply:
For whatever reason, the indentation on my IDE looks fine.